src/test/java/com/github/dockerjava/core/util/CertificateUtilsTest.java

```diff
@@ -0,0 +1,28 @@
+package com.github.dockerjava.core.util;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+public class CertificateUtilsTest {
```

Comment:
I think you need also the case when two files doesn't exist because it boolean operation and you tested it only with single absent file and "all exists".